server/server.js

```diff
@@ -1,8 +1,24 @@
 const express = require('express');
+const { createProxyMiddleware } = require('http-proxy-middleware');
 const path = require('path');
+const bodyParser = require('body-parser');
+require('newrelic');
 
-const app = express();
 const PORT = 3000;
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use('/products', createProxyMiddleware({
+  target: 'http://localhost:3002',
+  changeOrigin: true,
+}));
```

Comment:
This is how you proxy incoming url to target service. All `/products/*` will be forwarded to `http://localhost:3002/products/*`